src/index.ts

```diff
@@ -46,6 +47,8 @@ export const plugin: PluginFunction<Config> = async (schema, documents, configFr
 
   const interfaceTypes = Object.values(schema.getTypeMap()).filter(isInterfaceType);
 
+  const [mappedInterfaceTypes, unmappedInterfaceTypes] = interfaceTypes.partition((t) => isMappedType(t, config));
```

Comment:
Should this `mappedInterfaceTypes` being used somewhere?

Reply:
i guess not, initially i thought it would